index.html

```diff
@@ -1,16 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Andriana's Sweet Obsessions</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body class="main-section">
+    <header class="banner">
+        <nav class="hidden-menu">
+            <div class="menu-container">
+                <input type="checkbox"/>
+                <span class="material-symbols-outlined">menu</span>
+            <ul class="navbar">
+                <li><a href=".general">Home</a></li>
+                <li><a href=".cake-pics">Cakes</a></li>               
+                <li><a href="#">Order In</a></li>
+                <li><a href="#">Lessons</a></li>
+                <li><a href=".contact-icon">Contact Us</a></li>
```

Comment:
Is there a reason you are setting the values of your hrefs to classes you have? e.g. `href=".general"`